stratum/hal/bin/tdi/dpdk/dpdk_main.cc

```diff
@@ -139,11 +140,14 @@ ::util::Status Main(absl::Notification* ready_sync,
 
   auto auth_policy_checker = AuthPolicyChecker::CreateInstance();
 
+  TargetOptions target_options;
+  target_options.allowPipelineOverwrite = false;
+  target_options.pushUpdatedChassisConfig = false;
```

Comment:
Most likely, we will have to set similar values for ES2K target. I will check with SDE team once and get back to you. This can be taken as a separate PR